File: test/samples/lists/lists.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.Arrays;

class Main {
    /**
     * @param n the first list's size
     * @param listInt a list containing ints
     * @param size an other size
     * @param listChar a list of char
     * @param string a string
     * @param listString4 a list of strings of size 4
     * @param listListString2 a list of list of strings of size 2 of size 2 of size 2
     * @param matrix a matrix of int
     */
    static void lists(int n, int[] listInt, int size, char[] listChar, String string, String[] listString4, String[][] listListString2, int[][] matrix) {
        /* TODO Aren't these lists beautifull? */
    }

    public static void main(String[] args) throws java.io.IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        int n = Integer.parseInt(reader.readLine());
        int[] listInt = Arrays.stream(reader.readLine().split(" ")).mapToInt(Integer::parseInt).toArray();
        int size = Integer.parseInt(reader.readLine());
        char[] listChar = reader.readLine().toCharArray();
        String string = reader.readLine();
        String[] listString4 = new String[size];
        for (int i = 0; i < size; ++i) {
            listString4[i] = reader.readLine();
        }
        String[][] listListString2 = new String[2][];
        for (int i = 0; i < 2; ++i) {
            listListString2[i] = new String[2];
            for (int j = 0; j < 2; ++j) {
                listListString2[i][j] = reader.readLine();
            }
        }
        int[][] matrix = new int[size][];
        for (int i = 0; i < size; ++i) {
            matrix[i] = Arrays.stream(reader.readLine().split(" ")).filter(x -> !x.isEmpty()).mapToInt(Integer::parseInt).toArray();
        }

        lists(n, listInt, size, listChar, string, listString4, listListString2, matrix);
    }
}

File: test/samples/lists/lists.py
def lists(n, list_int, size, list_char, string, list_string4, list_list_string2, matrix):
    """
    :param n: the first list's size
    :type n: int
    :param list_int: a list containing ints
    :type list_int: List[int]
    :param size: an other size
    :type size: int
    :param list_char: a list of char
    :type list_char: List[str]
    :param string: a string
    :type string: str
    :param list_string4: a list of strings of size 4
    :type list_string4: List[str]
    :param list_list_string2: a list of list of strings of size 2 of size 2 of size 2
    :type list_list_string2: List[List[str]]
    :param matrix: a matrix of int
    :type matrix: List[List[int]]
    """
    # TODO Aren't these lists beautifull?
    pass


if __name__ == '__main__':
    n = int(input())
    list_int = list(map(int, input().split()))
    size = int(input())
    list_char = list(input())
    string = input()
    list_string4 = [input() for _ in range(size)]
    list_list_string2 = [[input() for _ in range(2)] for _ in range(2)]
    matrix = [list(map(int, input().split())) for _ in range(size)]
    lists(n, list_int, size, list_char, string, list_string4, list_list_string2, matrix)
